README.md

```diff
@@ -136,6 +136,40 @@ To make this project runnable via IntelliJ, do the following:
   is configured according to your needs
 
 
+## How to Run Tests
+
+### Test controls
+
+The project uses the `.sbtrc` [(link)](https://www.scala-sbt.org/1.x/docs/Best-Practices.html#.sbtrc) file to manage and control the execution of integration tests. The Integration tests are the ones with the tag `Integration test.` Below are the commands configured in the `.sbtrc` file to facilitate different testing scenarios:
+
+### Run Unit Tests
+Use the `test` command to execute all tests, skipping all Integration tests. 
+```
+sbt "project agent" ++2.13.11 test "project model" ++2.13.11 test
+```
+- with alias in .sbtrc:
+```
+alias test=; testOnly -- -l IntegrationTest
+
+# `-l IntegrationTest`: This switch excludes all tests labeled as `IntegrationTest,` allowing only only executing unit tests.
```

Comment:
```suggestion
# `-l IntegrationTest`: This switch excludes all tests labeled as `IntegrationTest,` allowing only executing unit tests.
```

Reply:
Fixed in commit - 3be6f06d762e44390d6720fe10fe1e2f6d30a3d0.